sdks/java/harness/src/test/java/org/apache/beam/fn/harness/logging/BeamFnLoggingClientTest.java

```diff
@@ -78,6 +78,7 @@
 
```

Comment:
It is best to write a test rule like https://github.com/apache/beam/blob/master/runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/testing/RestoreDataflowLoggingMDC.java and use it in the test instead of relying on everyone doing the appropriate clean-up.

Reply:
Done. created RestoreBeamFnLoggingMDC and RestoreBeamFnLoggingMDCTest